docker/pypi/wmagent/Dockerfile

```diff
@@ -48,16 +33,6 @@ RUN rm -f /etc/mysql/mariadb.conf.d/50-server.cnf
 # Add WMA_USER to sudoers
 RUN echo "${WMA_USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
-# Add all deployment needed directories
-ADD bin $WMA_DEPLOY_DIR/bin
```

Comment:
@todor-ivanov by the way, why is the manage script under the bin directory? It should have been WMA_MANAGE_DIR, if you wanted to keep a similar structure to what we have now.